lib/stdlib/src/erl_lint.erl

```diff
@@ -529,10 +534,9 @@ used_vars(Exprs, BindingsList) ->
 		  ({V,_Val}, Vs0) -> [{V,{bound,unused,[]}} | Vs0]
 	       end, [], BindingsList),
     Vt = orddict:from_list(Vs),
-    {Evt,_St} = exprs(set_file(Exprs, "nofile"), Vt, start()),
-    {ok, foldl(fun({V,{_,used,_}}, L) -> [V | L];
-                  (_, L) -> L
-	       end, [], Evt)}.
+    St0 = (start())#lint{fun_used_vars=maps:new()},
+    {_Evt,St1} = exprs(Exprs, Vt, St0),
+    St1#lint.fun_used_vars.
```

Comment:
This changed the return type spec from `{ok, map()}` to just `map()`, hence some (hacky, just like `power_shell`) software started to fail.

Reply:
I see. It was private API but I can figure something out if the Erlang/OTP team wants to bring it back.

Reply:
It is private API and my laziness, I forked `erl_eval` instead of making a small upstream fix (which I will do now).
I already pushed a fixed power_shell, so unless there are other usages of this function in the wild, I'd rather keep this change as is.